app/controllers/user_controllers.js

```diff
@@ -1,21 +1,87 @@
 'use strict';
 
 const logger = require('../logger'),
-  User = require('../models').Users;
+  bcrypt = require('bcryptjs'),
+  validator = require('validator'),
+  errors = require('../errors'),
+  User = require('../models').Users,
+  saltRounds = 10;
+
+const validateUser = user => {
+  let errorMsg = '';
+  // Name validation (Not null or empty)
+  if (!user.firstName || user.firstName.length <= 0) {
+    // First name error
+    errorMsg += 'First name cannot be empty.\n';
```

Comment:
Maybe you can do a list with error messages, instead of a large string formatted